content/2015.12.hacker-news.rst

```diff
@@ -0,0 +1,145 @@
+Hacker News answers
+###################
+
+:date: 2015-12-15
+:slug: hackernews-feedback
+:lang: en
+:url: hackernews-feedback
+:summary: Where we answer a set of questions that arised on a Hacker News
+          thread.
+
+Today, `Kinto was featured on Hackernews
+<https://news.ycombinator.com/item?id=10733164>`_. We take this as a great
+milestone for the project, even if it also comes with some sort of pressure and
+stress :)
+
+Since a few questions emerged from the thread there, we thought it'd
+appropriate to provide some answers here.
+
+What does this name mean?
+=========================
+
+Kinto-un is the name of the flying nimbus in Dragon ball. It's said that
+"the cloud can only be ridden by someone with a pure heart". Oh, well.
+    
+Maintaining a service is hard.
+==============================
+
+The main benefit of Firebase and Parse is that you don't have to run them.
+
+That is true. We want to make it very easy to deploy / use. We can even
+imagine to allow developers to run a Kinto instance in one-click on
+DigitalOcean, Heroku or Bitnami.
```

Comment:
Should we add a link to what we've started on Alwaysdata: https://github.com/Natim/auto-kinto-alwaysdata ?